src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestClient.java

```diff
@@ -22,21 +20,10 @@
    */
   SnowflakeStreamingIngestChannel openChannel(OpenChannelRequest request);
 
-  /**
-   * Close all the channels in the client, which will make sure all the data in all channels is
-   * committed before closing
-   *
-   * @return a completable future which will be completed when all the channels are closed
-   */
-  CompletableFuture<Void> close();
-
```

Comment:
Should we keep this in case customers use it in a non-Autocloseable context?

Reply:
if it's non-Autocloseable, they can still call the `close()` function explicitly, right? Did I understand correct what you mean?

Reply:
Yeah.